### ISSUE TITLE
Directories shown with a \ postfix when run on Linux

### ISSUE THREAD

**Issue by pa-jberanek** [gardebring/PowerColorLS#8]
When used on Linux directories are shown with a '\\' postfix rather than the more acceptable '/' suffix.

**Comment by pa-jberanek** [comment]
```
PS /tmp> mkdir test
PS /tmp> cd test
PS /tmp/test> mkdir directory
PS /tmp/test> powercolorls
 directory\
PS /tmp/test>
```

**Comment by gardebring** [comment]
This is indeed the case. There is currently no check to see whether powercolorls is running in a windows or linux environment and it defaults to the windows way of displaying directories.
I'll see if I can get some time to fix that, not sure when.

**Comment by gardebring** [comment]
This was fixed in commit 606e1fe867b03b3bcd0f7f45a71806bb8b79d3f2. Please update to PowerColorLS 1.0.3 to see the fix in action.
